lib/theme_check/file_system_storage.rb

```diff
@@ -16,14 +16,14 @@ def path(relative_path)
     end
 
     def read(relative_path)
-      file(relative_path).read
+      file(relative_path).read(mode: 'rb')
```

Comment:
Actually on the read, it's to detect what they are using. So that if we use the rewriter we don't lose that info and print it in the original EOL.

On the write it's so that we do not translate what we have in the ruby string. In the ThemeFiles, we do this:

* On read, we detect what the eol is and store the content with the \r stripped.
* On write, we put back the \r if we found out there were some.

We do it ourselves, because otherwise Source::Buffer would do it for us and we'd lose information.